ecsmanage/management/commands/ecsmanage.py

```diff
@@ -52,10 +52,10 @@ def handle(self, *args, **options):
 
         url = (
             f"https://console.aws.amazon.com/ecs/home?region={aws_region}#"
-            f"/clusters/{cluster_name}/tasks/{task_id}/details"
+            f"/clusters/{cluster_name}/tasks/{task_id}/details"  # NOQA
```

Comment:
I added these because these lines were getting marked as too long by `flake8` for specific version of Python (3.4 and 3.5), `despite max-line-length=88`. It remains unclear why.